### ISSUE TITLE
Backup Plan Idempotent Issue 

### ISSUE THREAD

**Issue by ss-sierra-cedar** [terraform-providers/terraform-provider-aws#13903]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version

Terraform v0.12.24
+ provider.archive v1.3.0
+ provider.aws v2.67.0
+ provider.null v2.1.2
+ provider.random v2.2.1
+ provider.template v2.1.2

### Affected Resource(s)

* aws_backup_plan
* aws_backup_selection

### Terraform Configuration Files

```
resource "aws_backup_plan" "efs_backup_plan_nprd" {
  name = "efs_backup_plan"

rule {
    rule_name         = "efs_backup_rule"
    target_vault_name = "efs_vault"
    schedule          = "cron(0 12 * * ? *)"
}
lifecycle {
      delete_after = "7"
}
}

resource "aws_backup_selection" "efs_backup_selection_nprd" {
  iam_role_arn = var.backup_restore_role_arn
  name         = "efs-backup-selection"
  plan_id      = aws_backup_plan.efs_backup_plan_nprd.id
  depends_on   = [aws_backup_plan.efs_backup_plan_nprd]
  resources = [
    "arn:aws:elasticfilesystem:${data.aws_region.current.name}:${data.aws_caller_identity.current.account_id}:file-system/${aws_efs_file_system.efs-ecs-1.id}",
    "arn:aws:elasticfilesystem:${data.aws_region.current.name}:${data.aws_caller_identity.current.account_id}:file-system/${aws_efs_file_system.efs-ecs-2.id}",
  ]
}
```

### Debug Output

Error: error reading Backup Selection: InvalidParameterValueException: Cannot find Backup plan with ID 303ed902-8915-46a1-b0c8-492be1b38d4f
{
  RespMetadata: {
    StatusCode: 400,
    RequestID: "e4b51b6a-13ca-4a7c-a01b-3dcb6b22b83c"
  },
  Code_: "ERROR_2304",
  Context: "303ed902-8915-46a1-b0c8-492be1b38d4f",
  Message_: "Cannot find Backup plan with ID 303ed902-8915-46a1-b0c8-492be1b38d4f"
}

### Expected Behavior

After deleting Backup Plan and Backup Selection via the console. The Backup Plan and Backup Selection were expected to  be recreated via terraform and update the state file. 

### Actual Behavior

Backup Plan failed to recreate and Backup Selection is not able to find the backup plan associated with the resource. 

### Steps to Reproduce

1. `Delete Backup Plan and Selection in the aws console`
2. `terraform apply`

**Comment by ewbankkit** [comment]
@ss-sierra-cedar Thanks for raising this issue.
I have opened https://github.com/terraform-providers/terraform-provider-aws/pull/13945 to fix this bug.

**Comment by bflad** [comment]
The fix for this has been merged and will release with version 2.69.0 of the Terraform AWS Provider, later this week. 👍
